src/features/metadata-instance-editor/Instances.js

```diff
@@ -14,27 +14,43 @@ type Props = {
         cascadingPolicy?: MetadataCascadingPolicyData,
         rawData: Object,
     ) => void,
+    selectedTemplateKey?: string,
+    templateFilters?: Set<string>,
 };
 
-const Instances = ({ isCascadingPolicyApplicable = false, editors = [], onModification, onRemove, onSave }: Props) =>
+const Instances = ({
+    isCascadingPolicyApplicable = false,
+    editors = [],
+    onModification,
+    onRemove,
+    onSave,
+    selectedTemplateKey,
+    templateFilters,
+}: Props) =>
     editors.map<React.Element<typeof Instance>>(
-        ({ isDirty = false, instance, hasError = false, template }: MetadataEditor) => (
-            <Instance
-                canEdit={instance.canEdit}
-                cascadePolicy={instance.cascadePolicy}
-                data={instance.data}
-                hasError={hasError}
-                id={instance.id}
-                isCascadingPolicyApplicable={isCascadingPolicyApplicable}
-                isDirty={isDirty}
-                isOpen={editors.length === 1}
-                key={`${instance.id}-${template.templateKey}`}
-                onModification={onModification}
-                onSave={onSave}
-                onRemove={onRemove}
-                template={template}
-            />
-        ),
+        ({ isDirty = false, instance, hasError = false, template }: MetadataEditor) => {
+            const { templateKey } = template;
+            // Open the included template by default, and only display the included fields in the template
+            const isOpen = templateKey === selectedTemplateKey || (!selectedTemplateKey && editors.length === 1);
```

Comment:
Can we retain the prior `isOpen` behavior, even if a `selectedTemplateKey` is provided?

```
isOpen={editors.length === 1 || templateKey === selectedTemplateKey}
```